packages/browser-wallet/src/popup/shared/utils/transaction-history-types.ts

```diff
@@ -0,0 +1,61 @@
+/**
+ * The interface for the result of a query to get historical transactions for
+ * an account, e.g. from the wallet proxy.
+ */
+export interface TransactionHistoryResult {
+    /** full indicates that the number of transactions received is equal to the limit given, i.e. it indicates that there could be additional transactions available. */
+    full: boolean;
+    transactions: BrowserWalletTransaction[];
+}
+
+export enum TransactionStatus {
+    /** On-chain but with an error */
+    Failed = 'failed',
+    /** On-chain and successful */
+    Finalized = 'finalized',
+}
+
+export enum TransactionType {
```

Comment:
Duplicate of AccountTransactionType from SDK?

Reply:
I need to add `ConfigureBaker` to the SDK and then I can do this. So I will wait for us to have a release with that included, and then update the pull request with the change.